src/native/fs/fs_napi.cpp

```diff
@@ -77,6 +92,27 @@ struct Entry
     uint8_t type;
 };
 
+struct gpfsRequest_t{
+    gpfsFcntlHeader_t header;
+    gpfsGetSetXAttr_t payload;
+    char buffer[GPFS_XATTR_BUFFER_SIZE];
+};
+
+void build_gpfs_get_ea_request(gpfsRequest_t* reqP) {
+    int nameLen = strlen(GPFS_XATTR_FULL_NAME);
```

Comment:
Since this is a literal string constant, you don't need to use strlen(), and can simply use sizeof().
However, notice that sizeof returns the entire memory length including the null terminator, so it will be 1 more than strlen.
```suggestion
    int nameLen = sizeof(GPFS_XATTR_FULL_NAME) - 1; // sizeof counts the memory including the last null char
```